include/SFML/Audio/SoundRecorder.h

```diff
@@ -132,5 +134,58 @@ CSFML_AUDIO_API sfBool sfSoundRecorder_isAvailable(void);
 ////////////////////////////////////////////////////////////
 CSFML_AUDIO_API void sfSoundRecorder_setProcessingInterval(sfSoundRecorder* soundRecorder, sfTime interval);
 
+////////////////////////////////////////////////////////////
+/// \brief Get a list of the names of all availabe audio capture devices
+///
+/// This function returns an array of strings, containing
+/// the names of all availabe audio capture devices.
+///
+/// \param count Pointer to a variable that will be filled with the number of modes in the array
+///
+/// \return An array of strings containing the names
+///
+////////////////////////////////////////////////////////////
+CSFML_AUDIO_API const char** sfSoundRecorder_getAvailableDevices(size_t* count);
+
+////////////////////////////////////////////////////////////
+/// \brief Get the name of the default audio capture device
+///
+/// This function returns the name of the default audio
+/// capture device. If none is available, an empty string
+/// is returned.
```

Comment:
Wouldn't NULL be more suitable for the default device? In C it is easier to test a NULL pointer than an empty string.